spack-environment/codeserver/spack.yml

```diff
@@ -5,7 +5,7 @@
 spack:
   # add package specs to the `specs` list
   specs:
-  - code-server@4.12.0
+  - code-server@4.96.4
```

Comment:
Interesting that this is the latest version available in spack, since [v4.96.4](https://github.com/coder/code-server/releases/tag/v4.96.4) released back in January. The latest version on github is [v4.103.1](https://github.com/coder/code-server/releases/tag/v4.103.1).

Reply:
Yeah, I think that's more recent than what we were using before. The [Spack packages repo](https://packages.spack.io/package.html?name=code-server) says that `code-server` is looking for a maintainer, so I don't think anyone is dedicated to its upkeep. We could update the package definition ourselves, or install it another way, if we wanted to be more up to date.